### PR TITLE
Fixes for example

### PR DESCRIPTION
**Pull request by sorru94**
Fixes the example, adding the required configuration options and doing some generic cleanup.
Also bumps up the SDK version to `1.1.2`.

### CODE REVIEW

File: examples/edgehog_app/main/include/example_task.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef EXAMPLE_TASK_H
+#define EXAMPLE_TASK_H
```
Comment (harlem88) [inline examples/edgehog_app/main/include/example_task.h:21]: Leave a new line between the license and code.
```suggestion
 */

#ifndef EXAMPLE_TASK_H
#define EXAMPLE_TASK_H
```

File: examples/edgehog_app/main/include/wifi_cfg.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef WIFI_CFG_H
```
Comment (harlem88) [inline examples/edgehog_app/main/include/wifi_cfg.h:20]: same above
```suggestion

#ifndef WIFI_CFG_H
```

File: examples/edgehog_app/main/src/wifi_cfg.c [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "wifi_cfg.h"
```
Comment (harlem88) [inline examples/edgehog_app/main/src/wifi_cfg.c:20]:
```suggestion
 */
 
#include "wifi_cfg.h"
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: examples/edgehog_app/main/src/example_task.c [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "example_task.h"
```
[the remark this answers is not in the document]
Comment (sorru94) [reply examples/edgehog_app/main/src/example_task.c]: This conflicts with the used configuration for `clang-format`.

**Review by harlem88 — APPROVED** [review]

**Pull request merged**